django/db/migrations/state.py

```diff
@@ -451,15 +451,16 @@ def flatten_bases(model):
             if model._default_manager.use_in_migrations:
                 default_manager = copy.copy(model._default_manager)
                 default_manager._set_creation_counter()
+                managers.append((force_text(default_manager.name), default_manager))
 
             # If the default manager doesn't have `use_in_migrations = True`,
             # shim a default manager so another manager isn't promoted in its
             # place.
-            else:
+            elif model._default_manager.name != 'objects':
```

Comment:
I don't think it's about `objects` as a name for the base manager / default manager. It's about if there's >= 1 manager that's used in migrations.